package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-router-proxy-loader",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
You're totally right, plus it does affect behaviour external to the project so it should be a minor at least.